docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  web:
+    build: .
+    environment:
+      NODE_ENV:
+      MONGODB_URI: mongodb://mongo:27017/nodegoat
+    command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
+    ports:
+      - "4000:4000"
+
+  mongo:
+    image: mongo:4.4
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- b45d3ec5ad351ba84236c484844d7cfe -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Improper Privilege Management
_Low CWE-269_
The Docker Compose file sets the `no-new-privileges` attribute to `false` or omits it as the default value is `false`. This doesn't restrict the container from acquiring additional privileges via SUID or SGID bits. This allows privilege escalation, meaning a user can set the security context of the container and can perform root-level operations.



### How to fix
Potential mitigations include the following:
  1. Very carefully manage the setting, management, and handling of privileges. Explicitly manage trust zones in the software.
  2. Follow the principle of least privilege when assigning access rights to entities in a software system.
  3. Consider following the principle of separation of privilege. Require multiple conditions to be met before permitting access to a system resource.